v2/CHANGES

```diff
@@ -29,14 +29,13 @@
       ordering, perhaps taking availability changes into account.
 - [ ] Audit yarpcerrors, remove the ones that are no longer needed.
       - [ ] add staticcheck for v2/internal/internalyarpcerror/yarpcerrors.go from `local.mk`
-- [ ] Construct PRs with retroactive continuity.
-- [ ] Nit: scrape name pid and change to just id throughout.
+- [x] Nit: scrape name id and change to just id throughout.
 - [ ] Audit docs
       - [ ] gRPC
       - [ ] TChannel
       - [ ] HTTP
 - [ ] InvokeUnaryHandler: reduce package name noise at the call site by changing to a Do(ctx) method on UnaryInvokeRequest
-- [ ] support StreamHandlers in EncodingHandlerSpec
+- [ ] support StreamHandlers in EncodingHandlerSpec (punt)
```

Comment:
Perhaps we can change this to merely, [x] Figure out the role of streams in EncodingHandlerSpec: streaming handlers will only apply at the transport level.